src/components/scopedVariables/types.tsx

```diff
@@ -5,6 +5,16 @@ export enum ReadFileAs {
     ARRAY_BUFFER = 'arrayBuffer',
 }
 
+export enum FileView {
+    YAML = 'yaml',
+    SAVED = 'saved',
+}
+
+export interface SavedVariablesViewI {
+    scopedVariables: any
```

Comment:
Please define the type if possible

Reply:
Right now schema, is subject to change and variables can have any value.